besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -1347,6 +1347,33 @@ public void ethStatsContactOptionCannotBeUsedWithoutEthStatsServerProvided() {
             "The `--Xethstats-contact` requires ethstats server URL to be provided. Either remove --Xethstats-contact or provide an url (via --Xethstats=nodename:secret@host:port)");
   }
 
+  @Test
+  public void dnsEnabledOptionIsParsedCorrectly() {
+    TestBesuCommand besuCommand = parseCommand("--Xdns-enabled", "true");
+
+    assertThat(besuCommand.getEnodeDnsConfiguration().dnsEnabled()).isEqualTo(true);
```

Comment:
nit; `isTrue()` and `isFalse()` can be nicer to read then `isEqualTo(true)` and `isEqualTo(false)`

Reply:
Done